### ISSUE TITLE
Comment on weights

### ISSUE THREAD

**Issue by davelab6** [simoncozens/silson#2]
In general your Light master is too heavy, so your instances are very close together and can't really be used to create typography hierarchy.

**Comment by simoncozens** [comment]
How are we looking now? (I've only done the key letters so far.) To me it's starting to look like a proper family, but I am unsure about some of the curves of the thin master.

![screen shot 2016-02-10 at 15 36 24](https://cloud.githubusercontent.com/assets/106728/12938787/1aaf829a-d00c-11e5-8e37-2efe262f1a9d.png)

[test-masters.pdf](https://github.com/simoncozens/silson/files/124206/test-masters.pdf)

**Comment by davelab6** [comment]
Yes, the 's' is pince-ing. I suggest to look at FF Kava / Kaffeesatz

**Comment by simoncozens** [comment]
I'll have a fiddle with the /s - the superelliptic/squircular corners are a design element of the font (see /d counter, /o counter, etc.), so I don't want to lose them, but I agree the shape isn't quite right.

What I was _expecting_ comment on was the shoulder of the /d and the weight distribution in the /a. (And to a lesser extent /e bowl.) But maybe they're OK if you didn't notice anything...
